### ISSUE TITLE
UX: In Toolbar (only) Rename "Modules" to "Functionality"

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#209]
The Problem: New users do not know to click on 'Modules' to extend their site.

Discussion on d.o:
- https://drupal.org/node/1167458
- https://drupal.org/node/1164760

**Comment by sutibun** [comment]
Functionality is such a weird long word.

I still remember the first time I played with Backdrop some time ago. **The word "Functionality" on the admin bar made me stop, look at it twice, and scratch my head.** Only when I hovered over and saw "modules" did I get it.

Would "Features" work better? I wonder if people are more familiar with the word "Features" than "Functionality"

- From a salesman, "this product is the best. Let me give you a rundown of all its features"
- "Buy MS Windows because it has the killer feature"
- "What features does this car have?"

Or, could just be me not encountering "functionality" often. I just found it so out of place.
